### ISSUE TITLE
Error on new version 2.3.0

### ISSUE THREAD

**Issue by Lughino** [dlmanning/gulp-sass#473]
Hi,
I get this error when I launch the task:

```javascript
/var/www/test/node_modules/node-sass/lib/index.js:180
  options.includePaths = (options.includePaths || []).join(path.delimiter);
                                                      ^

TypeError: (options.includePaths || []).join is not a function
    at getOptions (/var/www/test/node_modules/node-sass/lib/index.js:180:55)
    at Object.module.exports.render (/var/www/test/node_modules/node-sass/lib/index.js:267:13)
    at DestroyableTransform._transform (/var/www/test/node_modules/gulp-sass/index.js:138:25)
    at DestroyableTransform.Transform._read (/var/www/test/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:159:10)
    at DestroyableTransform.Transform._write (/var/www/test/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:147:83)
    at doWrite (/var/www/test/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:313:64)
    at writeOrBuffer (/var/www/test/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:302:5)
    at DestroyableTransform.Writable.write (/var/www/test/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:241:11)
    at DestroyableTransform.ondata (/var/www/test/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:531:20)
    at emitOne (events.js:77:13)
    at DestroyableTransform.emit (events.js:169:7)
    at readableAddChunk (/var/www/test/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:198:18)
    at DestroyableTransform.Readable.push (/var/www/test/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:157:10)
    at DestroyableTransform.Transform.push (/var/www/test/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:123:32)
    at afterTransform (/var/www/test/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:79:51)
    at TransformState.afterTransform (/var/www/test/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:58:12)
```

with gulp-sass version 2.2.0 there is no problem.

**Comment by xzyfer** [comment]
Thanks for report @Lughino. I believe I have a fix in https://github.com/dlmanning/gulp-sass/pull/474. Waiting on confirmation from the person who removed this code.

**Comment by xzyfer** [comment]
A work around for now is setting the `includePaths: []` option.

**Comment by bigjollygiant14** [comment]
Same issue for me. @extend and @import aren't working anymore. On demo day too :(. Going to try the include paths for now.

**Comment by timhettler** [comment]
In previous versions `includePaths` accepted a String

**Comment by xzyfer** [comment]
I've released v2.3.1 which restores the old behaviour.
